File: DailyFlash Solutions/DailyFlash Solutions/07-DailyFlash_Solutions/21_Jan_Solutions_One/Python/tableof.py
"""*Program 4: Write a Program that accepts an integer from user and print table
of that number.
Input: 4
Output: 4 8 12 16 20 24 28 36 40"""


num = input("Enter the number ")

for itr in range(1,11):
    print(itr*num)
    

File: DailyFlash Solutions/DailyFlash Solutions/07-DailyFlash_Solutions/21_Jan_Solutions_One/Java/TableOf.java
/*Program 4: Write a Program that accepts an integer from user and print table
of that number.
Input: 4
Output: 4 8 12 16 20 24 28 36 40*/

import java.io.*;

class TableOf{
	
	public static void main(String args[])throws IOException{
		
		int num;

		BufferedReader br  = new BufferedReader(new InputStreamReader(System.in));
		
		num = Integer.parseInt(br.readLine());

		for(int itr = 1 ;itr <= 10 ; itr++){

			System.out.printf("%d \n",itr*num);
		}	

	}	
}	
